hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1429,6 +1430,16 @@ private void initializeRatisDirs(OzoneConfiguration conf) throws IOException {
       }
       OmUtils.createOMDir(omRatisDirectory);
 
+      String omStorageDir = conf.get(OMConfigKeys.OZONE_OM_RATIS_STORAGE_DIR);
+      String scmStorageDir =
+          conf.get(ScmConfigKeys.OZONE_SCM_HA_RATIS_STORAGE_DIR);
+      if (omStorageDir.equals(scmStorageDir)) {
```

Comment:
Please add a null check as if they are unset they would be null and by default they are different paths so there shouldn't be an issue.
```suggestion
      if (omStorageDir!=null && scmStorageDir!=null && omStorageDir.equals(scmStorageDir)) {
```

Reply:
Done

Reply:
actually there is one problem here, not very practically likely but it still exists.
If I set omStorageDir = defaultPath of scm , then we won't throw the exception here as the scmStorageDir is not defined but actually they would be same.
The right way to do this would be to 
``` java
omStorageDir = OzoneManagerRatisUtils.getOMRatisDirectory(conf);
scmStorageDir = RatisUtil.getRatisStorageDir(conf); 
```
and then compare.

Reply:
Done